kanalony-storage-access/src/main/scala/kanalony/storage/generator/CassandraCqlGenerator.scala

```diff
@@ -0,0 +1,47 @@
+package kanalony.storage.generator
+
+import java.io.File
+
+import kanalony.storage.utils.fs
+
+/**
+ * Created by elad.benedict on 3/27/2016.
+ */
+object CassandraCqlGenerator {
+
+  def main (args: Array[String]) {
+
+    val keySpaceName = "kanalony_user_activity"
+    var res = List(s"CREATE KEYSPACE IF NOT EXISTS ${keySpaceName} WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'}  AND durable_writes = true;")
+
+    val tableNamePlaceholder = "%TABLE_NAME%"
+    val columnDefsPlaceholder = "%COLUMN_DEFS%"
+    val partitionKeyColsPlaceholder = "%PARTITION_KEY_COLS%"
+    val clusteringColsPlaceholder = "%CLUSTERING_KEY_COLS%"
+    val timeColPlaceholder = "%TIME_COL%"
+
+    val tableCreationTemplate = s"""CREATE TABLE IF NOT EXISTS ${keySpaceName}.%TABLE_NAME% (
+                                  |    %COLUMN_DEFS%
+                                  |    PRIMARY KEY ((%PARTITION_KEY_COLS%), %CLUSTERING_KEY_COLS%)
+                                  |) WITH CLUSTERING ORDER BY (%TIME_COL% DESC);"""
```

Comment:
Can you please add 2 optional params to the TablesMetadata properties:
default_time_to_live: default=0   (on minutely tables should be 15552000 which is 180 days; on 10sec tables should be 604800 which is 7 days)
gc_grace_seconds: default=43200 (12 hours)

cc: @olampert